about.html

```diff
@@ -1,13 +1,198 @@
+                        </div>
+                        <div class="offcanvas-body">
+                            <ul class="navbar-nav justify-content-center flex-grow-1 pe-3">
+                                <li class="nav-item text-color1">
+                                    <a class="nav-link" aria-current="page" href="#" data-bs-dismiss="offcanvas"
+                                    aria-label="Close">About</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#" data-bs-dismiss="offcanvas"
+                                        aria-label="Close">Program</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">Join</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">Sponsor</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" href="#">News</a>
+                                </li>
+                                <li class="nav-item ms-5">
+                                    <a class="navbar-brand d-none d-lg-block campaign" href="#">Meshkut</a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </nav>
+        </div>
+        <div class="container">
+            <div class="row">
+                <div class="col head_content mb-5">
+                    <div class="mt-4 ms-1 active rounded">
+                        <h5 class="text-color1 text-color2 d-md-none">"Hello! Sharing world"</h5>
+                        <div id="demo-text" class=" fw-bolder px-1 costom-div">MESHKUT EDUCATIONAL <br>SOCIETY
+                            FOR 2023</div>
+                        <P class="text5 tx-custom-black white-border bg-white fs-small  px-4 py-3 mt-4 mb-4">Meshkut
+                            Educational Society
+                            is a non-profit educational institute focused on providing access to to QoS.
+                            Meshkut
+                            Educational
+                            Society is an education management company based in Afghanistan.
+                            Meshkut
+                            Educational Society
+                            is a non-profit educational institute focused on providing access to to QoS. <span class="must1">
+                            Society is an education management company based in Afghanistan.
+                            Meshkut
+                            Educational Society
+                            is a non-profit educational institute focused on providing access to to QoS.</span></P>
```

Comment:
Kindly update the hero section in your about page to match the behance design better🙏

Your design 
<img width="946" alt="image" src="https://user-images.githubusercontent.com/37116322/233183136-78bb3cf6-f885-46d5-8be2-b4f11aafee64.png">

Behance design
<img width="554" alt="image" src="https://user-images.githubusercontent.com/37116322/233183239-4b3c25c8-cac2-44f2-ae7f-53e644ea6c1c.png">

A few things to note:
- Please update the main title to have a much larger font size and font weight and be center aligned
- Please add more top and bottom padding to the section